field/goldilocks/goldilocks_plonky2.go

```
@@ -111,7 +114,8 @@ func NegF(x GoldilocksField) GoldilocksField {
 }
 
 func SampleF() GoldilocksField {
-	return GoldilocksField(rand.Uint64N(ORDER))
+	rng, _ := rand.Int(rand.Reader, ORDER_BIG)
```

I am not sure if we want to ignore this error